hudi-client/hudi-java-client/src/test/java/org/apache/hudi/table/action/commit/TestJavaCopyOnWriteActionExecutor.java

```diff
@@ -480,4 +478,76 @@ public void testBulkInsertRecords(String bulkInsertMode) throws Exception {
     records1.addAll(records2);
     return records1;
   }
+
+  @Test
+  public void testHoodieConcatHandle() throws Exception {
```

Comment:
Wondeing if this can be rewritten more nicely using the TestTable abstraction. cc @xushiyan .